### ISSUE TITLE
Does not build with python3.9

### ISSUE THREAD

**Issue by didiercolens** [scoder/acora#22]
FYI, this package does not install after python3.8:

```bash
$ docker run -it --rm python:3.9.1-buster /bin/bash
root@3053158056ee:/# pip3 install acora
Collecting acora
  Downloading acora-2.2.tar.gz (210 kB)
     |████████████████████████████████| 210 kB 4.4 MB/s 
Building wheels for collected packages: acora
  Building wheel for acora (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /usr/local/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-0um7_g35/acora_ebe641025ee34edfbf51454eee319dc3/setup.py'"'"'; __file__='"'"'/tmp/pip-install-0um7_g35/acora_ebe641025ee34edfbf51454eee319dc3/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-zj45cgqs
       cwd: /tmp/pip-install-0um7_g35/acora_ebe641025ee34edfbf51454eee319dc3/
  Complete output (142 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-3.9
  creating build/lib.linux-x86_64-3.9/acora
  copying acora/__init__.py -> build/lib.linux-x86_64-3.9/acora
  copying acora/_acora.py -> build/lib.linux-x86_64-3.9/acora
  running build_ext
  building 'acora._acora' extension
  creating build/temp.linux-x86_64-3.9
  creating build/temp.linux-x86_64-3.9/acora
  gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/local/include/python3.9 -c acora/_acora.c -o build/temp.linux-x86_64-3.9/acora/_acora.o
  acora/_acora.c: In function ‘__Pyx_modinit_type_init_code’:
  acora/_acora.c:10468:43: error: ‘PyTypeObject’ {aka ‘struct _typeobject’} has no member named ‘tp_print’; did you mean ‘tp_dict’?
     __pyx_type_5acora_6_acora__MachineState.tp_print = 0;
                                             ^~~~~~~~
                                             tp_dict
  acora/_acora.c:10479:38: error: ‘PyTypeObject’ {aka ‘struct _typeobject’} has no member named ‘tp_print’; did you mean ‘tp_dict’?
     __pyx_type_5acora_6_acora__Machine.tp_print = 0;
                                        ^~~~~~~~
                                        tp_dict
  acora/_acora.c: In function ‘__Pyx_ParseOptionalKeywords’:
  acora/_acora.c:11096:21: warning: ‘_PyUnicode_get_wstr_length’ is deprecated [-Wdeprecated-declarations]
                       (PyUnicode_GET_SIZE(**name) != PyUnicode_GET_SIZE(key)) ? 1 :
                       ^
  In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                   from /usr/local/include/python3.9/Python.h:97,
                   from acora/_acora.c:16:
  /usr/local/include/python3.9/cpython/unicodeobject.h:446:26: note: declared here
   static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObject *op) {
                            ^~~~~~~~~~~~~~~~~~~~~~~~~~
  acora/_acora.c:11096:21: warning: ‘PyUnicode_AsUnicode’ is deprecated [-Wdeprecated-declarations]
                       (PyUnicode_GET_SIZE(**name) != PyUnicode_GET_SIZE(key)) ? 1 :
                       ^
  In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                   from /usr/local/include/python3.9/Python.h:97,
                   from acora/_acora.c:16:
  /usr/local/include/python3.9/cpython/unicodeobject.h:580:45: note: declared here
   Py_DEPRECATED(3.3) PyAPI_FUNC(Py_UNICODE *) PyUnicode_AsUnicode(
                                               ^~~~~~~~~~~~~~~~~~~
  acora/_acora.c:11096:21: warning: ‘_PyUnicode_get_wstr_length’ is deprecated [-Wdeprecated-declarations]
                       (PyUnicode_GET_SIZE(**name) != PyUnicode_GET_SIZE(key)) ? 1 :
                       ^
  In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                   from /usr/local/include/python3.9/Python.h:97,
                   from acora/_acora.c:16:
  /usr/local/include/python3.9/cpython/unicodeobject.h:446:26: note: declared here
   static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObject *op) {
                            ^~~~~~~~~~~~~~~~~~~~~~~~~~
  acora/_acora.c:11096:21: warning: ‘_PyUnicode_get_wstr_length’ is deprecated [-Wdeprecated-declarations]
                       (PyUnicode_GET_SIZE(**name) != PyUnicode_GET_SIZE(key)) ? 1 :
                       ^
  In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                   from /usr/local/include/python3.9/Python.h:97,
                   from acora/_acora.c:16:
  /usr/local/include/python3.9/cpython/unicodeobject.h:446:26: note: declared here
   static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObject *op) {
                            ^~~~~~~~~~~~~~~~~~~~~~~~~~
  acora/_acora.c:11096:21: warning: ‘PyUnicode_AsUnicode’ is deprecated [-Wdeprecated-declarations]
                       (PyUnicode_GET_SIZE(**name) != PyUnicode_GET_SIZE(key)) ? 1 :
                       ^
  In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                   from /usr/local/include/python3.9/Python.h:97,
                   from acora/_acora.c:16:
  /usr/local/include/python3.9/cpython/unicodeobject.h:580:45: note: declared here
   Py_DEPRECATED(3.3) PyAPI_FUNC(Py_UNICODE *) PyUnicode_AsUnicode(
                                               ^~~~~~~~~~~~~~~~~~~
  acora/_acora.c:11096:21: warning: ‘_PyUnicode_get_wstr_length’ is deprecated [-Wdeprecated-declarations]
                       (PyUnicode_GET_SIZE(**name) != PyUnicode_GET_SIZE(key)) ? 1 :
                       ^
  In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                   from /usr/local/include/python3.9/Python.h:97,
                   from acora/_acora.c:16:
  /usr/local/include/python3.9/cpython/unicodeobject.h:446:26: note: declared here
   static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObject *op) {
                            ^~~~~~~~~~~~~~~~~~~~~~~~~~
  acora/_acora.c:11112:25: warning: ‘_PyUnicode_get_wstr_length’ is deprecated [-Wdeprecated-declarations]
                           (PyUnicode_GET_SIZE(**argname) != PyUnicode_GET_SIZE(key)) ? 1 :
                           ^
  In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                   from /usr/local/include/python3.9/Python.h:97,
                   from acora/_acora.c:16:
  /usr/local/include/python3.9/cpython/unicodeobject.h:446:26: note: declared here
   static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObject *op) {
                            ^~~~~~~~~~~~~~~~~~~~~~~~~~
  acora/_acora.c:11112:25: warning: ‘PyUnicode_AsUnicode’ is deprecated [-Wdeprecated-declarations]
                           (PyUnicode_GET_SIZE(**argname) != PyUnicode_GET_SIZE(key)) ? 1 :
                           ^
  In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                   from /usr/local/include/python3.9/Python.h:97,
                   from acora/_acora.c:16:
  /usr/local/include/python3.9/cpython/unicodeobject.h:580:45: note: declared here
   Py_DEPRECATED(3.3) PyAPI_FUNC(Py_UNICODE *) PyUnicode_AsUnicode(
                                               ^~~~~~~~~~~~~~~~~~~
  acora/_acora.c:11112:25: warning: ‘_PyUnicode_get_wstr_length’ is deprecated [-Wdeprecated-declarations]
                           (PyUnicode_GET_SIZE(**argname) != PyUnicode_GET_SIZE(key)) ? 1 :
                           ^
  In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                   from /usr/local/include/python3.9/Python.h:97,
                   from acora/_acora.c:16:
  /usr/local/include/python3.9/cpython/unicodeobject.h:446:26: note: declared here
   static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObject *op) {
                            ^~~~~~~~~~~~~~~~~~~~~~~~~~
  acora/_acora.c:11112:25: warning: ‘_PyUnicode_get_wstr_length’ is deprecated [-Wdeprecated-declarations]
                           (PyUnicode_GET_SIZE(**argname) != PyUnicode_GET_SIZE(key)) ? 1 :
                           ^
  In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                   from /usr/local/include/python3.9/Python.h:97,
                   from acora/_acora.c:16:
  /usr/local/include/python3.9/cpython/unicodeobject.h:446:26: note: declared here
   static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObject *op) {
                            ^~~~~~~~~~~~~~~~~~~~~~~~~~
  acora/_acora.c:11112:25: warning: ‘PyUnicode_AsUnicode’ is deprecated [-Wdeprecated-declarations]
                           (PyUnicode_GET_SIZE(**argname) != PyUnicode_GET_SIZE(key)) ? 1 :
                           ^
  In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                   from /usr/local/include/python3.9/Python.h:97,
                   from acora/_acora.c:16:
  /usr/local/include/python3.9/cpython/unicodeobject.h:580:45: note: declared here
   Py_DEPRECATED(3.3) PyAPI_FUNC(Py_UNICODE *) PyUnicode_AsUnicode(
                                               ^~~~~~~~~~~~~~~~~~~
  acora/_acora.c:11112:25: warning: ‘_PyUnicode_get_wstr_length’ is deprecated [-Wdeprecated-declarations]
                           (PyUnicode_GET_SIZE(**argname) != PyUnicode_GET_SIZE(key)) ? 1 :
                           ^
  In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                   from /usr/local/include/python3.9/Python.h:97,
                   from acora/_acora.c:16:
  /usr/local/include/python3.9/cpython/unicodeobject.h:446:26: note: declared here
   static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObject *op) {
                            ^~~~~~~~~~~~~~~~~~~~~~~~~~
  acora/_acora.c: In function ‘__Pyx_decode_c_bytes’:
  acora/_acora.c:13555:9: warning: ‘PyUnicode_FromUnicode’ is deprecated [-Wdeprecated-declarations]
           return PyUnicode_FromUnicode(NULL, 0);
           ^~~~~~
  In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                   from /usr/local/include/python3.9/Python.h:97,
                   from acora/_acora.c:16:
  /usr/local/include/python3.9/cpython/unicodeobject.h:551:42: note: declared here
   Py_DEPRECATED(3.3) PyAPI_FUNC(PyObject*) PyUnicode_FromUnicode(
                                            ^~~~~~~~~~~~~~~~~~~~~
  error: command '/usr/bin/gcc' failed with exit code 1
  ----------------------------------------
  ERROR: Failed building wheel for acora
  Running setup.py clean for acora
Failed to build acora
Installing collected packages: acora
    Running setup.py install for acora ... error
    ERROR: Command errored out with exit status 1:
     command: /usr/local/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-0um7_g35/acora_ebe641025ee34edfbf51454eee319dc3/setup.py'"'"'; __file__='"'"'/tmp/pip-install-0um7_g35/acora_ebe641025ee34edfbf51454eee319dc3/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-1phktx0j/install-record.txt --single-version-externally-managed --compile --install-headers /usr/local/include/python3.9/acora
         cwd: /tmp/pip-install-0um7_g35/acora_ebe641025ee34edfbf51454eee319dc3/
    Complete output (142 lines):
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.9
    creating build/lib.linux-x86_64-3.9/acora
    copying acora/__init__.py -> build/lib.linux-x86_64-3.9/acora
    copying acora/_acora.py -> build/lib.linux-x86_64-3.9/acora
    running build_ext
    building 'acora._acora' extension
    creating build/temp.linux-x86_64-3.9
    creating build/temp.linux-x86_64-3.9/acora
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/local/include/python3.9 -c acora/_acora.c -o build/temp.linux-x86_64-3.9/acora/_acora.o
    acora/_acora.c: In function ‘__Pyx_modinit_type_init_code’:
    acora/_acora.c:10468:43: error: ‘PyTypeObject’ {aka ‘struct _typeobject’} has no member named ‘tp_print’; did you mean ‘tp_dict’?
       __pyx_type_5acora_6_acora__MachineState.tp_print = 0;
                                               ^~~~~~~~
                                               tp_dict
    acora/_acora.c:10479:38: error: ‘PyTypeObject’ {aka ‘struct _typeobject’} has no member named ‘tp_print’; did you mean ‘tp_dict’?
       __pyx_type_5acora_6_acora__Machine.tp_print = 0;
                                          ^~~~~~~~
                                          tp_dict
    acora/_acora.c: In function ‘__Pyx_ParseOptionalKeywords’:
    acora/_acora.c:11096:21: warning: ‘_PyUnicode_get_wstr_length’ is deprecated [-Wdeprecated-declarations]
                         (PyUnicode_GET_SIZE(**name) != PyUnicode_GET_SIZE(key)) ? 1 :
                         ^
    In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                     from /usr/local/include/python3.9/Python.h:97,
                     from acora/_acora.c:16:
    /usr/local/include/python3.9/cpython/unicodeobject.h:446:26: note: declared here
     static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObject *op) {
                              ^~~~~~~~~~~~~~~~~~~~~~~~~~
    acora/_acora.c:11096:21: warning: ‘PyUnicode_AsUnicode’ is deprecated [-Wdeprecated-declarations]
                         (PyUnicode_GET_SIZE(**name) != PyUnicode_GET_SIZE(key)) ? 1 :
                         ^
    In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                     from /usr/local/include/python3.9/Python.h:97,
                     from acora/_acora.c:16:
    /usr/local/include/python3.9/cpython/unicodeobject.h:580:45: note: declared here
     Py_DEPRECATED(3.3) PyAPI_FUNC(Py_UNICODE *) PyUnicode_AsUnicode(
                                                 ^~~~~~~~~~~~~~~~~~~
    acora/_acora.c:11096:21: warning: ‘_PyUnicode_get_wstr_length’ is deprecated [-Wdeprecated-declarations]
                         (PyUnicode_GET_SIZE(**name) != PyUnicode_GET_SIZE(key)) ? 1 :
                         ^
    In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                     from /usr/local/include/python3.9/Python.h:97,
                     from acora/_acora.c:16:
    /usr/local/include/python3.9/cpython/unicodeobject.h:446:26: note: declared here
     static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObject *op) {
                              ^~~~~~~~~~~~~~~~~~~~~~~~~~
    acora/_acora.c:11096:21: warning: ‘_PyUnicode_get_wstr_length’ is deprecated [-Wdeprecated-declarations]
                         (PyUnicode_GET_SIZE(**name) != PyUnicode_GET_SIZE(key)) ? 1 :
                         ^
    In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                     from /usr/local/include/python3.9/Python.h:97,
                     from acora/_acora.c:16:
    /usr/local/include/python3.9/cpython/unicodeobject.h:446:26: note: declared here
     static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObject *op) {
                              ^~~~~~~~~~~~~~~~~~~~~~~~~~
    acora/_acora.c:11096:21: warning: ‘PyUnicode_AsUnicode’ is deprecated [-Wdeprecated-declarations]
                         (PyUnicode_GET_SIZE(**name) != PyUnicode_GET_SIZE(key)) ? 1 :
                         ^
    In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                     from /usr/local/include/python3.9/Python.h:97,
                     from acora/_acora.c:16:
    /usr/local/include/python3.9/cpython/unicodeobject.h:580:45: note: declared here
     Py_DEPRECATED(3.3) PyAPI_FUNC(Py_UNICODE *) PyUnicode_AsUnicode(
                                                 ^~~~~~~~~~~~~~~~~~~
    acora/_acora.c:11096:21: warning: ‘_PyUnicode_get_wstr_length’ is deprecated [-Wdeprecated-declarations]
                         (PyUnicode_GET_SIZE(**name) != PyUnicode_GET_SIZE(key)) ? 1 :
                         ^
    In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                     from /usr/local/include/python3.9/Python.h:97,
                     from acora/_acora.c:16:
    /usr/local/include/python3.9/cpython/unicodeobject.h:446:26: note: declared here
     static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObject *op) {
                              ^~~~~~~~~~~~~~~~~~~~~~~~~~
    acora/_acora.c:11112:25: warning: ‘_PyUnicode_get_wstr_length’ is deprecated [-Wdeprecated-declarations]
                             (PyUnicode_GET_SIZE(**argname) != PyUnicode_GET_SIZE(key)) ? 1 :
                             ^
    In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                     from /usr/local/include/python3.9/Python.h:97,
                     from acora/_acora.c:16:
    /usr/local/include/python3.9/cpython/unicodeobject.h:446:26: note: declared here
     static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObject *op) {
                              ^~~~~~~~~~~~~~~~~~~~~~~~~~
    acora/_acora.c:11112:25: warning: ‘PyUnicode_AsUnicode’ is deprecated [-Wdeprecated-declarations]
                             (PyUnicode_GET_SIZE(**argname) != PyUnicode_GET_SIZE(key)) ? 1 :
                             ^
    In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                     from /usr/local/include/python3.9/Python.h:97,
                     from acora/_acora.c:16:
    /usr/local/include/python3.9/cpython/unicodeobject.h:580:45: note: declared here
     Py_DEPRECATED(3.3) PyAPI_FUNC(Py_UNICODE *) PyUnicode_AsUnicode(
                                                 ^~~~~~~~~~~~~~~~~~~
    acora/_acora.c:11112:25: warning: ‘_PyUnicode_get_wstr_length’ is deprecated [-Wdeprecated-declarations]
                             (PyUnicode_GET_SIZE(**argname) != PyUnicode_GET_SIZE(key)) ? 1 :
                             ^
    In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                     from /usr/local/include/python3.9/Python.h:97,
                     from acora/_acora.c:16:
    /usr/local/include/python3.9/cpython/unicodeobject.h:446:26: note: declared here
     static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObject *op) {
                              ^~~~~~~~~~~~~~~~~~~~~~~~~~
    acora/_acora.c:11112:25: warning: ‘_PyUnicode_get_wstr_length’ is deprecated [-Wdeprecated-declarations]
                             (PyUnicode_GET_SIZE(**argname) != PyUnicode_GET_SIZE(key)) ? 1 :
                             ^
    In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                     from /usr/local/include/python3.9/Python.h:97,
                     from acora/_acora.c:16:
    /usr/local/include/python3.9/cpython/unicodeobject.h:446:26: note: declared here
     static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObject *op) {
                              ^~~~~~~~~~~~~~~~~~~~~~~~~~
    acora/_acora.c:11112:25: warning: ‘PyUnicode_AsUnicode’ is deprecated [-Wdeprecated-declarations]
                             (PyUnicode_GET_SIZE(**argname) != PyUnicode_GET_SIZE(key)) ? 1 :
                             ^
    In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                     from /usr/local/include/python3.9/Python.h:97,
                     from acora/_acora.c:16:
    /usr/local/include/python3.9/cpython/unicodeobject.h:580:45: note: declared here
     Py_DEPRECATED(3.3) PyAPI_FUNC(Py_UNICODE *) PyUnicode_AsUnicode(
                                                 ^~~~~~~~~~~~~~~~~~~
    acora/_acora.c:11112:25: warning: ‘_PyUnicode_get_wstr_length’ is deprecated [-Wdeprecated-declarations]
                             (PyUnicode_GET_SIZE(**argname) != PyUnicode_GET_SIZE(key)) ? 1 :
                             ^
    In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                     from /usr/local/include/python3.9/Python.h:97,
                     from acora/_acora.c:16:
    /usr/local/include/python3.9/cpython/unicodeobject.h:446:26: note: declared here
     static inline Py_ssize_t _PyUnicode_get_wstr_length(PyObject *op) {
                              ^~~~~~~~~~~~~~~~~~~~~~~~~~
    acora/_acora.c: In function ‘__Pyx_decode_c_bytes’:
    acora/_acora.c:13555:9: warning: ‘PyUnicode_FromUnicode’ is deprecated [-Wdeprecated-declarations]
             return PyUnicode_FromUnicode(NULL, 0);
             ^~~~~~
    In file included from /usr/local/include/python3.9/unicodeobject.h:1026,
                     from /usr/local/include/python3.9/Python.h:97,
                     from acora/_acora.c:16:
    /usr/local/include/python3.9/cpython/unicodeobject.h:551:42: note: declared here
     Py_DEPRECATED(3.3) PyAPI_FUNC(PyObject*) PyUnicode_FromUnicode(
                                              ^~~~~~~~~~~~~~~~~~~~~
    error: command '/usr/bin/gcc' failed with exit code 1
    ----------------------------------------
ERROR: Command errored out with exit status 1: /usr/local/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-0um7_g35/acora_ebe641025ee34edfbf51454eee319dc3/setup.py'"'"'; __file__='"'"'/tmp/pip-install-0um7_g35/acora_ebe641025ee34edfbf51454eee319dc3/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-1phktx0j/install-record.txt --single-version-externally-managed --compile --install-headers /usr/local/include/python3.9/acora Check the logs for full command output.
```

This is only seen from 3.9 (latest 3.8.x image installs it fine).

**Comment by JelleZijlstra** [comment]
I ran into this too and found that `python3.9 -m pip install 'git+https://github.com/scoder/acora.git'` works, presumably because that makes pip build the package with a newer version of Cython. A release that is built with recent Cython should automatically work for 3.9.

**Comment by scoder** [comment]
Fix released as acora 2.3.
